general/web_apis.md

```diff
@@ -0,0 +1,40 @@
+# Best practices when using Web APIs
+
+## Local Storage
+
+When writing implementations using `localStorage` first we must understand that `localStorage` is prone to evaluating as `undefined`.
```

Comment:
"prone" makes it sounds like it happens all the time. I actually think it's pretty rare that somebody would turn it off in their prefs. Maybe just say "may evaluate as `undefined`)".